### PR TITLE
Key mapping for alpha and symbolic keys

### PR DESCRIPTION
**Pull request by chatrapathi06**
Key mapping for alpha and symbolic keys for Keyboard Keys in  X11 and LibWpe

### CODE REVIEW

**Review by pprabhu-ng — CHANGES_REQUESTED** [review]
@chatrapathi06  its better to keep the RNS key names similar to WPE .
Ensure spelling checks for the keys.

File: ReactSkia/sdk/RNSKeyCodeMapping.h [hunk 1]
```diff
@@ -39,11 +42,122 @@ typedef enum rnsKey{
...
+    RNS_KEY_Control_L,
+    RNS_KEY_Control_R,
+    /*Special char*/
+    RNS_KEY_Tab,
+    RNS_KEY_Grave,
+    RNS_KEY_Asciitilde,
+    RNS_KEY_Exclam,
+    RNS_KEY_At,
+    RNS_KEY_Numbersign,
+    RNS_KEY_Dolar,
```
Comment (pprabhu-ng) [inline ReactSkia/sdk/RNSKeyCodeMapping.h]: should be RNS_KEY_Dollar ?

File: ReactSkia/sdk/RNSKeyCodeMapping.h [hunk 2]
```diff
@@ -39,11 +42,122 @@ typedef enum rnsKey{
...
+    RNS_KEY_Period,
+    RNS_KEY_Comma,
+    RNS_KEY_Question,
+    RNS_KEY_Apostrophe,
+    RNS_KEY_Quotedbl,
+    RNS_KEY_Semicolon,
+    RNS_KEY_Colon,
+    RNS_KEY_Bar,
+    RNS_KEY_BackSlash,
+    RNS_KEY_Braceleft,
```
Comment (pprabhu-ng) [inline ReactSkia/sdk/RNSKeyCodeMapping.h]: Braceleft/right to be defined similar to other keys

File: ReactSkia/sdk/RNSKeyCodeMapping.h [hunk 3]
```diff
@@ -96,6 +210,117 @@ static std::string RNSKeyMap[RNS_KEY_End]={
...
+     * Arthematic Opps
+     */
+    "-",
+    "=",
+    "%",
+    "+",
+    ">",
+    "<",
+
+    "UnKnown"
```
Comment (pprabhu-ng) [inline ReactSkia/sdk/RNSKeyCodeMapping.h]: need to maintain consistency in defining the strings for the keys.
few keynames are camelcase, few are pascal case.

**Review by pprabhu-ng — CHANGES_REQUESTED** [review]

File: packages/react-native-skia/modules/tvEventHandler/tvEventHandler.js [hunk 4]
```diff
@@ -20,6 +20,7 @@ class SimpleViewApp extends React.Component {
     console.log('APP: TVEventHandler ',this._tvEventHandler);
     this._tvEventHandler.enable(this, function(cmp, evt) {
       console.log('APP: TV Key event received: ', evt);
+      console.log('APP: TV Key event received:  [evt.eventType]',  evt.eventType);
```
Comment (munezbn) [inline packages/react-native-skia/modules/tvEventHandler/tvEventHandler.js]: Now we can also print eventKeyAction, tag and target

File: rns_shell/platform/graphics/x11/X11KeyMap.cpp [hunk 5]
```diff
@@ -1,6 +1,7 @@
 #include <X11/Xutil.h>
 #include "ReactSkia/sdk/RNSKeyCodeMapping.h"
 #include "WindowX11.h"
+#include "ReactSkia/utils/RnsLog.h"
```
Comment (munezbn) [inline rns_shell/platform/graphics/x11/X11KeyMap.cpp]: can we move up before line 3

**Review by pprabhu-ng — APPROVED** [review]

**Review by munezbn — APPROVED** [review]

**Pull request merged**